IndicoV2.Abstractions/Jobs/IJobsClient.cs

```diff
@@ -22,16 +21,7 @@ public interface IJobsClient
         /// <param name="jobId">Job's Id</param>
         /// <param name="cancellationToken"><c><see cref="CancellationToken"/></c> for handling cancellation of asynchronous operations.</param>
         /// <returns>Job result</returns>
-        [Obsolete("Use generic version of this method.")]
-        Task<JToken> GetResultAsync(string jobId, CancellationToken cancellationToken = default);
-
-        /// <summary>
-        /// Gets Job's result
-        /// </summary>
-        /// <param name="jobId">Job's Id</param>
-        /// <param name="cancellationToken"><c><see cref="CancellationToken"/></c> for handling cancellation of asynchronous operations.</param>
-        /// <returns>Job result</returns>
-        Task<TResult> GetResultAsync<TResult>(string jobId, CancellationToken cancellationToken = default);
+        Task<string> GetResultAsync(string jobId, TimeSpan checkInterval, CancellationToken cancellationToken = default);
```

Comment:
We shouldn't change the return type or add new non-optional parameters to a function if we can avoid it.

Reply:
reverted